docs/site/sidebars/lb4_sidebar.yml

```diff
@@ -242,6 +242,13 @@ children:
       url: Database-migrations.html
       output: 'web, pdf'
 
+  - title: 'Using TypeORM'
```

Comment:
any reason we need a parent item instead of having "How to use TypeORM with LoopBack" as the top level?

Reply:
There will be more articles under it later. Eg: "How to create custom repositories", etc.